schemas/aerleon-definitions.schema.json

```diff
@@ -1,18 +1,22 @@
 {
+  "$id": "urn:aerleon:schemas:aerleon-definitions:1.0.0",
   "$schema": "http://json-schema.org/draft-07/schema#",
   "$comment": "https://aerleon.readthedocs.io/en/latest/reference/naming/",
-  "title": "Aerleon Definition File",
+  "title": "Aerleon Network & Service Definitions",
+  "description": "define named networks and services",
   "type": "object",
   "additionalProperties": false,
   "properties": {
     "networks": {
-      "description": "An Aerleon definition file defines named networks and services which can be referenced by Aerleon Policy files. Networks are lists of IP addresses or CIDR IP address ranges. Services are lists of port/protocol pairs (e.g. tcp/80) and can include port ranges.\nDefinition files must include a \"networks\" section, a \"services\" section, or both.\nNetworks and services may use composition in their definitions. Include another definition by creating an item containing the \"name\" property.",
+      "title": "Network Definition Section",
+      "description": "Policy files can reference the networks in this section by name. A network is a list of IP addresses or CIDR IP address ranges and may contain other networks.",
       "type": "object",
       "additionalProperties": { "$ref": "#/$defs/networkDefinition" },
       "propertyNames": { "$ref": "#/$defs/token" }
     },
     "services": {
-      "description": "An Aerleon definition file defines named networks and services which can be referenced by Aerleon Policy files. Networks are lists of IP addresses or CIDR IP address ranges. Services are lists of port/protocol pairs (e.g. tcp/80) and can include port ranges.\nDefinition files must include a \"networks\" section, a \"services\" section, or both.\nNetworks and services may use composition in their definitions. Include another definition by creating an item containing the \"name\" property.",
+      "title": "Service Definition Section",
+      "description": "Policy files can reference the services in this section by name. A service is a list of port/protocol pairs (e.g. tcp/80) and can include port ranges. A service can include other services.",
```

Comment:
The example here is backwards both in terms of how you refer to it `port/protocol` and how it is used `HTTP:: 80/tcp`. You should consider switching it in the example here to `80/tcp`
You may also wish to include a port range example.

Reply:
You can axe the second part. Just realized this is only for the YAML spec so the `HTTP:: 80/tcp` makes no sense.